client/state/ui/guided-tours/contexts.js

```diff
@@ -133,6 +134,16 @@ export const hasSelectedSiteDefaultSiteTitle = state => {
 };
 
 /**
+ * Returns true if user has just pasted something from Google Docs.
+ *
+ * @param {Object} state Global state tree
+ * @return {Boolean} True if user has just pasted something from Google Docs, false otherwise.
+ */
+export const hasUserPastedFromGoogleDocs = state => {
+	return getLastAction( state ).type === EDITOR_PASTE_EVENT_FROM_GDOCS;
```

Comment:
Thanks for the attention to detail, Miguel. I ended up implementing a different approach, but I've written some tests to reflect this conversation.